src/cmd/simplesslerrorhandler.cpp

```diff
@@ -27,7 +27,7 @@ bool SimpleSslErrorHandler::handleErrors(QList<QSslError> errors, const QSslConf
         return false;
     }
 
-    foreach (QSslError error, errors) {
+    for (QSslError error : errors) {
```

Comment:
`QSslError` is not a scalar type, so better go for a const ref here (`const auto &` ideally).
Also, `errors` not being const, the container will detach, so either the parameter is converted to a const ref (my preference but more invasive in the context of this patch) or it needs to be `qAsConst(errors)`.